lib/transform/config.js

```diff
@@ -32,7 +32,7 @@ function transformConfig(config, dictionary, platformName) {
 
   // The platform can define either a transformGroup or an array
   // of transforms. If given a transformGroup that doesn't exist,
-  // it will throw an error so the user is aware the transformGroup doesn't
+  // it will throw an error and the user is aware the transformGroup doesn't
```

Comment:
> // it will throw an error and the user is aware the transformGroup doesn't

// it will throw an error to make the user aware that the transformGroup doesn't